src/utils/create-auto-suggest.ts

```diff
@@ -4,19 +4,15 @@ function createAutosuggestComponent(config: PluginSettings) {
   const w3wComponent = document.createElement('what3words-autosuggest');
 
   w3wComponent.setAttribute('variant', 'inherit');
-  // w3wComponent.setAttribute(
-  //   "headers",
-  //   JSON.stringify({
-  //     "X-W3W-Plugin":
-  //       `what3words-Wordpress/${what3wordsConfig?.version} (` +
-  //       [
-  //         `PHP/${what3wordsConfig?.php_version}`,
-  //         `WordPress/${what3wordsConfig?.wp_version}`,
-  //         `ecwid/${what3wordsConfig?.wc_version}`,
-  //       ].join(" ") +
-  //       ")",
-  //   })
-  // );
+  w3wComponent.setAttribute(
+    'headers',
+    JSON.stringify({
+      'X-W3W-Plugin':
+        'what3words-ecwid(' +
+        [`ecwid/${window.Ecwid?.storefrontVersion}`].join(' ') +
+        ')',
```

Comment:
I believe the format for this header is incorrect. Please check [WP Autosuggest]()
```suggestion
      'X-W3W-Plugin':
        `what3words-ecwid/${this application version goes here} (` + [
          `ecwid/${window.Ecwid?.storefrontVersion}`
        ].join(' ') + ')',
```